calliope/test/common/test_model/group_constraints.yaml

```diff
@@ -0,0 +1,55 @@
+model:
+        name: Test model for group constraint mechanism
+        timeseries_data_path: 'timeseries_data'
```

Comment:
For this and all other models, probably worth just having it run over one day, to minimise optimisation time, unless there's some additional information that comes from running over a longer time period...

Reply:
Gave it a try just now and that kills 50% of test time. So that's certainly a good idea.